mite/test.py

```diff
@@ -72,19 +73,20 @@ def http_spy(journey):
         spy = _InterceptHttp(_NewHttp(requests), ctx)
         await journey(spy, *args)
         return {'http': requests}
+
     return wrapper
 
 
 async def run_single_journey(config, journey, datapool=None):
     messages = []
 
-    async def _send(message, **kwargs):
+    def _send(message, **kwargs):
```

Comment:
How was this working?

Reply:
It's probably left over from my attempt to introduce more asynchronicity in the zmq methods, which I had to back out when it didn't exactly work.  This code is to support unit tests external to mite, so it's not been exercised since then until now as I'm trying to resurrect the id-mite-nft unit test suite...